### ISSUE TITLE
[Article template] Using both microdata and JSON+ld which cause SEO duplicates

### ISSUE THREAD

**Issue by ludoboludo** [Shopify/dawn#1958]
**Describe the current behavior**

On `main-article.liquid` we currently have both microdata and JSON+ld used for structured data. 
The problem is that it's creating some duplication which isn't great for SEO reasons. 

Though microdata was used for accessibility reasons: https://scottvinkle.me/blogs/work/how-html-microdata-helps-with-accessibility

**Describe the expected behavior**

Which should only rely on one form of structured data. 

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 6

**Possible solution**

Use one over the other. Is keeping microdata on the article template ok ? While we keep JSON+ld else where or should have the same approach across the board 🤔 

**Additional context/screenshots**

**Comment by EllionGrey** [comment]
Hey @ludoboludo ! Thank you for looking into this. I've disabled the `JSON+ld` method in our merchants theme but they noticed that the microdata method doesn't appear to capture the `author` or `image` that exists: https://search.google.com/test/rich-results/result/r%2Farticles?id=ErVVUh_g2yOQZNX7szM4sQ

![image](https://user-images.githubusercontent.com/37596162/191296444-99bf4293-7d8f-4744-9be0-ea08e842268c.png)

Do we know if this would have a negative impact on their SEO score or is this data captured elsewhere by google? 

REF: 

* shop: https://app.shopify.com/services/internal/shops/63765774584
* Rich results link: https://search.google.com/test/rich-results
* Page for testing: https://luxurycbds.com/blogs/learn/a-brief-history-of-cbd

**Comment by EllionGrey** [comment]
Hey team! Just circling back to this one as my merchant is reaching out to request updates. Do we know where this sits in terms of priority? :)
